doc/sphinx-guides/source/installation/config.rst

```diff
@@ -1875,6 +1875,17 @@ To check the current value of ``:ShibAffiliationAttribute``:
 
 .. _:ComputeBaseUrl:
 
+:ShibAttributeCharacterSetConversionEnabled
++++++++++++++++++++++++++++++++++++++++++++
+
+It seems that Glassfish will interpret all Shibboleth attributes that come through AJP as ISO-8859-1, even if they where originally UTF-8.
+To circumvent that, we re-encode all received Shibboleth attributes manually as UTF-8 by default. 
+In the case you get garbled characters in Shibboleth-supplied fields (e.g. given name, surname, affiliation), you can disable this behaviour by setting ShibAttributeCharacterSetConversionEnabled to false:
+
+``curl -X PUT -d false http://localhost:8080/api/admin/settings/:ShibAttributeCharacterSetConversionEnabled``
+
+If you managed to get correct accented characters from shibboleth while this setting is _false_, please contact us with your Glassfish and Shibboleth configuration!
+
```

Comment:
@pallinger overall, these docs are looking great. They should be above ` .. _:ComputeBaseUrl:` which is an anchor. Also, since we're moving from Glassfish to Payara, if you could change "Glassfish" to "the application server" (or "your application server") as appropriate, it would be great.